app/src/main/java/com/github/mik629/android/fundamentals/App.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.mik629.android.fundamentals
+
+import android.app.Application
+import com.github.mik629.android.fundamentals.di.AppComponent
+import com.github.mik629.android.fundamentals.di.DaggerAppComponent
+import timber.log.Timber
+import javax.inject.Inject
+
+class App : Application() {
+
+    @Inject
+    lateinit var tree: Timber.Tree
+
+    override fun onCreate() {
+        super.onCreate()
+
+        appComponent = DaggerAppComponent.builder()
```

Comment:
use dagger.factory instead of builder

Reply:
Created an [issue](https://github.com/mik629/AndroidFundamentalsApp/issues/10)